docs/source/sfm.rst

```diff
@@ -817,9 +822,9 @@ Incremental SfM is generally considered to be more robust than global SfM
 methods; however, it requires many more instances of bundle adjustment (which
 is very costly) and so incremental SfM is not as efficient or scalable.
 
-To use the Incremental SfM pipeline, simple set the
+To use the Incremental SfM pipeline, simply set the
 ``reconstruction_estimator_type`` to
-``ReconstructionEstimatorType::INCREMENTAL``. There are many more options that
+``ReconstructionEstimatorType::INCREMENTAL``. There are much more options that
```

Comment:
It should stay as "many". The difference between "much" and "many" is that "much" refers to uncountable nouns and "many" refers to countable nouns. In this case, we can count the number of options so it should be "many".